src/main/java/com/joss/conductor/mobile/ConductorConfig.java

```diff
@@ -175,6 +186,12 @@ private void setProperty(String propertyName, String propertyValue) {
         while (envMatcher.find()) {
             String env = envMatcher.group(1);
             String val = System.getProperty(env);
+
+            // If the system property is not set '-Dsystem_property=<value` then look for an environment variable
```

Comment:
Looks like you're missing the right hand '>' char around the value placeholder w/in your comment.